### ISSUE TITLE
Enhanced Aria does not work in NVDA 2018.3Alpha builds

### ISSUE THREAD

**Issue by MarcoZehe** [jmdaweb/enhancedAria#1]
There are no settings in the configuration, and if I go to the EnhancedAria panel, I get this error logged:

DEBUGWARNING - Python warning (18:41:12.951):
C:\Users\marco\AppData\Roaming\nvda\addons\enhancedAria\globalPlugins\enhancedAria.py:75: DeprecationWarning: NewId() is deprecated
ERROR - unhandled exception (18:41:12.984):
Traceback (most recent call last):
  File "gui\settingsDialogs.pyc", line 2474, in onCategoryChange
  File "gui\settingsDialogs.pyc", line 569, in onCategoryChange
  File "gui\settingsDialogs.pyc", line 547, in _doCategoryChange
  File "gui\settingsDialogs.pyc", line 480, in _getCategoryPanel
  File "gui\settingsDialogs.pyc", line 247, in __init__
  File "C:\Users\marco\AppData\Roaming\nvda\addons\enhancedAria\globalPlugins\enhancedAria.py", line 158, in makeSettings
  File "C:\Users\marco\AppData\Roaming\nvda\addons\enhancedAria\globalPlugins\enhancedAria.py", line 75, in createSettings
wxAssertionError: C++ assertion "id == wxID_ANY || (id >= 0 && id < 32767) || (id >= wxID_AUTO_LOWEST && id <= wxID_AUTO_HIGHEST)" failed at ..\..\src\common\wincmn.cpp(372) in wxWindowBase::CreateBase(): invalid id value

The panel itself comes up blank, no checkboxes.

**Comment by jmdaweb** [comment]
Hello.

After installing the latest alpha build, Enhanced Aria settings are
displayed in the settings dialog, so I can't reproduce this issue. I
need more information.

Regards.

**Comment by jmdaweb** [comment]
All deprecated calls to wx.NewId() have been replaced with wx.ID_ANY, as described in the WXPython API documentation. If you continue having this problem, let me know and I'll reopen it.
